tests/plugins/test_koji_promote.py

```diff
@@ -961,13 +961,15 @@ def test_koji_promote_image_tags(self, tmpdir, os_env, additional_tags):
          None),
 
     ])
-    @pytest.mark.parametrize('has_config', [
-        True,
-        False,
+    @pytest.mark.parametrize(('has_config', 'is_autorebuild'), [
+        (True,
```

Comment:
The way `parametrize` is being used here is by providing two sets of values for the `has_config` and `is_autorebuild`. However, both sets have the same value: `(True, False)`. Meaning, in both cases, `has_config` is `True`, and `is_autorebuild` is `False`. Perhaps you meant this:

```python
@pytest.mark.parametrize('has_config', [True, False])
@pytest.mark.parametrize('is_autorebuild', [True, False])
def test_koji_promote_success(...
```